Lib/test/test_ast/test_ast.py

```diff
@@ -2303,15 +2303,14 @@ def get_load_const(self, tree):
         co = compile(tree, '<string>', 'exec')
         consts = []
         for instr in dis.get_instructions(co):
-            if instr.opname == 'LOAD_CONST' or instr.opname == 'RETURN_CONST':
+            if instr.opname in ('LOAD_CONST', 'LOAD_CONST_IMMORTAL'):
```

Comment:
Since `dis.get_instructions` is called without an `adaptive=True` arg, I think it's impossible to have a `LOAD_CONST_IMMORTAL` here?